cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -59,6 +59,24 @@ angular.module(PKG.name + '.commons')
       //     'data-label': 'Date'
       //   }
       // },
+      'datetime': {
```

Comment:
This doesn't need to be here right? Now that we have written all our components in react. We need to add this map to `AbstractWidgetFactory` right?